src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -87,6 +87,11 @@ void Terminal::CreateFromSettings(winrt::Microsoft::Terminal::Settings::ICoreSet
 {
     const COORD viewportSize{ Utils::ClampToShortMax(settings.InitialCols(), 1),
                               Utils::ClampToShortMax(settings.InitialRows(), 1) };
+
+    if (_suppressApplicationTitle)
+    {
+        _title = _startingTitle;
+    }

```

Comment:
er, this is not in the right place and will never trigger 😦 

It needs to be after line 98, `UpdateSettings`, because these values are never being set to anything otherwise..

Reply:
...oops...I should not have merged that huh... Sorry!

Reply:
I shouldn't have Approved with pending "please write new code" comments :smile: